src/mono/sample/wasm/blazor-frame/AppStart.cs

```diff
@@ -0,0 +1,16 @@
+using System.Runtime.InteropServices.JavaScript;
+
+namespace BrowserBench
+{
+        public partial class FrameApp
+        {
+            [JSImport("globalThis.frameApp.ReachedCallback")]
+            public static partial Task ReachedCallback();
+
+            [JSExport]
+            public static void ReachedManaged()
```

Comment:
Blazor measures "first time to UI" and does it by overrideing `OnAfterRender` and stop the measure from there. Wouldn't be convinient to have similar measurement? Or do you want explicitly exclude blazor rendering from the measurement?

Reply:
Good idea, I can add it in the next PR.

Reply:
I think it would be good to know both "reach managed" and "blazor first render" separately.
Make sure you are not measuring just the new server side rendering  :)